.changeset/great-plums-buy.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Why the empty changeset?

Reply:
These are not fields are not used and contain empty values so I don't think we need a changelog entry for them.